vendor/lib/lib_path.cmd

```diff
@@ -85,13 +85,13 @@ exit /b
 
     if /i "%position%" == "append" (
       if "!found!" == "0" (
-        echo "%path%"|%WINDIR%\System32\findstr >nul /I /R /C:";%find_query%\"$"
+        echo "%path%"|%WINDIR%\System32\findstr >nul /I /R /C:";%find_query%\"
         call :set_found
       )
       %lib_console% debug_output  :enhance_path "Env Var END PATH %find_query% - found=!found!"
     ) else (
       if "!found!" == "0" (
-        echo "%path%"|%WINDIR%\System32\findstr >nul /I /R /C:"^\"%find_query%;"
+        echo "%path%"|%WINDIR%\System32\findstr >nul /I /R /C:"%find_query%;"
```

Comment:
`^` at the beginning is part of a regex that says find this at the beginning of the string removing it removes the intention of this entire line of code.

Reply:
You're right - sorry.
Changing to /C:"^\%find_query%;" resolves the issue in the following screenshot.

![after-cexec_cmd-change](https://user-images.githubusercontent.com/4619155/90201582-a3feee80-de1e-11ea-8753-b0d68be2e3d0.png)